core/src/main/java/hudson/model/Computer.java

```diff
@@ -691,6 +691,7 @@ public boolean isTemporarilyOffline() {
      */
     @Deprecated
     public void setTemporarilyOffline(boolean temporarilyOffline) {
+        LOGGER.log(Level.WARNING, "deprecated function call {0}", new Throwable().getStackTrace());
```

Comment:
It makes no sense to inform the admin about something they almost certainly can't do anything about anyway (we have a few more, but generally not basic method invocations).

Reply:
I see. I want only see where it is used. In core it is no more used. I need the plugins too. Maybe we can just remove the function.